.claude/hooks/stop.py

```
@@ -0,0 +1,171 @@
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            "--chat", action="store_true", help="Copy transcript to chat.json"
+        )
+        args = parser.parse_args()
+
+        # Read JSON input from stdin
+        input_data = json.load(sys.stdin)
+
+        # Extract required fields
+        session_id = input_data.get("session_id", "")
+        stop_hook_active = input_data.get("stop_hook_active", False)
```

Ruff (0.12.2)

Local variable `stop_hook_active` is assigned to but never used

Remove assignment to unused variable `stop_hook_active`

(F841)